try.py

```diff
@@ -0,0 +1,71 @@
+import httpx
+
+from deepset_cloud_sdk.workflows.sync_client.files import (
+    DeepsetCloudFileBytes,
+    WriteMode,
+    upload,
+    upload_bytes,
+)
+
+api_key = "api_eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIwYjYwYzE2YS00NjJlLTRkNmMtYjg2ZS1mYTI3M2RhNGU2Nzh8NjI5NzY4NWVmMzdiNTIwMDY4NDU3ODMxIiwiZXhwIjoxNzE2NTA1MjAwLCJhdWQiOlsiaHR0cHM6Ly9hcGkuZGV2LmNsb3VkLmRwc3QuZGV2Il19.MjfWK8JtrkVpSLwb-wJ06vxkw2sQM3sB6Okd47PX3Cw"
```

Comment:
🔔 🔔 credentials detected => let's drop and invalidate the token

Reply:
oops!